modules/samples/producers/kafka/src/main/java/org/wso2/carbon/sample/kafka/KafkaUtil.java

```diff
@@ -48,16 +53,18 @@ public static String getMessageFilePath(String sampleNumber, String filePath, St
 		if (filePath != null && sampleNumber == null) {
 			resultingFilePath = filePath;
 		} else if (filePath == null && sampleNumber != null) {
-			String urlSplitter[] = url.split("/");
-			resultingFilePath = sampleFilPath.replace("sampleNumber", sampleNumber)+urlSplitter[urlSplitter.length-1]+fileExtension;
+			if(format.equalsIgnoreCase("csv")){
```

Comment:
Thanks @lafernando for pointing that out. I will fix it. I copy and pasted the above code from `JMSClientUtil.java` in the jms producer. We may need to update that as well. Perhaps in a separate PR?